src/router/gallery.js

```diff
@@ -78,6 +66,21 @@ module.exports = function createGalleryRouter(config) {
 
 	router.use(formidable.parse());
 
+	if (!(config.infoProxy || config.getInfoProxy)) {
```

Comment:
use atomic negations:
!config.infoProxy && !config.getInfoProxy

or it's even better to check if the getInfoProxy is a function.

Also, the error message should be more descriptive.